src/resources/js/components/Presentation.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+
+ <v-container>
+     <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+      <v-card
+      class="mx-auto"
+      max-width="344"
+      >
+      <v-list-item three-line>
+        <v-list-item-content>
+          <v-list-item-title class="headline mb-1">
+            Titre : {{ presentation.data.title }}
+          </v-list-item-title>
+          <v-list-item-subtitle>
+            Identifiant: {{ presentation.data.id }}
+          </v-list-item-subtitle>
```

Comment:
This field as no meaning for the users so it should not be show
And missing usefull field date

Reply:
ok